models/cms_hcc/staging/cms_hcc__stg_financial_pmpm__member_months.sql

```diff
@@ -0,0 +1,12 @@
+{{ config(
+     enabled = var('cms_hcc_enabled',var('financial_pmpm_enabled', var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))) | as_bool
+   )
+}}
```

Comment:
I would remove the var references for financial_pmpm and clinical. That could cause issues. We only want this model to run if the cms_hcc mart is enabled, claims, or all tuva marts.

```suggestion
{{ config(
     enabled = var('cms_hcc_enabled',var('claims_enabled',var('tuva_marts_enabled',False))) | as_bool
   )
}}
```

Reply:
Just confirming that the clinical var did cause issues with our [CI testing](https://github.com/tuva-health/tuva/actions/runs/9358423396), so we'll need to remove it.

Reply:
@sarah-tuva removed, left the financial_pmpm in as I saw that comment was deleted.